sdk/storage/azure-storage-blob/apilisting.txt

```diff
@@ -0,0 +1,591 @@
+class AccessPolicy(Model, object):
+	permission:permission
+	expiry:expiry
+	start:start
+
+	def __eq__(self, other): ...
+	def __init__(self, permission: permission=None, expiry: expiry=None, start: start=None): ...
+	def __ne__(self, other): ...
+	def __str__(self): ...
+	def as_dict(self, keep_readonly=True, key_transformer: function=<function attribute_transformer at 0x10e6a4510>) -> dict: ...
+	@classmethod
+	def deserialize(cls, data: str, content_type: str=None): ...
+	@classmethod
+	def enable_additional_properties_sending(cls): ...
+	@classmethod
+	def from_dict(cls, data: dict, key_extractors=None, content_type: str=None): ...
+	@classmethod
+	def is_xml_model(cls): ...
+	def serialize(self, keep_readonly: bool=False) -> dict: ...
+	def validate(self) -> list: ...
+
+class AccountPermissions(object):
```

Comment:
In the other languages, we appear to include `Sas` (or similar) as part of the type-name.  Should we do the same here and for the other "Permissions" types here?